h2o-algos/src/test/java/hex/tfidf/TfIdfPreprocessorTest.java

```diff
@@ -0,0 +1,49 @@
+package hex.tfidf;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+import water.TestUtil;
+import water.fvec.Frame;
+import water.fvec.TestFrameBuilder;
+import water.fvec.Vec;
+
+public class TfIdfPreprocessorTest extends TestUtil {
+
+    @BeforeClass()
+    public static void setup() { stall_till_cloudsize(1); }
+
+    @Test
+    public void testTfIdfPreprocessing() {
+        Frame fr = getSimpleTestFrame();
+        byte[] outputTypes = new byte[]{Vec.T_STR, Vec.T_NUM, Vec.T_NUM};
```

Comment:
This line causes the error (`DistributedException from /192.168.0.164:54321: 'NewChunk has type String, but the Vec is of type Numeric'`). The third type (word tokens) should be `Vec.T_STR`.

Reply:
Thank you. Careless mistake.